### ISSUE TITLE
propertyUnits fail on local variables

### ISSUE THREAD

**Issue by jwilsson** [lesshint/lesshint#82]
As first reported by @kokarn over at [lesshint/atom-linter-lesshint#1](https://github.com/lesshint/atom-linter-lesshint/issues/1).

It looks like variables are treated as properties but without an `ident` and therefore fails when trying to read a `ident` value from it. 

A simple check for an `ident` value should suffice.

**Comment by kokarn** [comment]
```less
.box-sizing (@type: border-box) {
    -webkit-box-sizing: @type;
    -moz-box-sizing:    @type;
    box-sizing:         @type;
}

* {
    .box-sizing();
}
```

**Comment by jwilsson** [comment]
This should've been fixed with #85.
